Sources/IRGen/Expression/IRAttemptExpression.swift

```diff
@@ -21,10 +21,13 @@ struct IRAttemptExpression {
       callName = IRWrapperFunction.prefixSoft + functionName
     }
 
-    let args: String = functionCall.arguments.map({ argument in
-      return IRExpression(expression: argument.expression, asLValue: false).rendered(functionContext: functionContext)
-    }).joined(separator: ", ")
+    let (preamble, args) = functionCall.arguments.reduce(
+      ("", ""), { (r, argument) in
```

Comment:
I think here it would make more sense to reduce over a `GeneratedCode` struct itself, with empty strings by default. Then you create a new one on each iteration